docs/docs/how-tos/multi-agent-network-functional.ipynb

```diff
@@ -0,0 +1,500 @@
+    "        Sign up for LangSmith to quickly spot issues and improve the performance of your LangGraph projects. LangSmith lets you use trace data to debug, test, and monitor your LLM apps built with LangGraph — read more about how to get started <a href=\"https://docs.smith.langchain.com\">here</a>. \n",
+    "    </p>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4a53f304-3709-4df7-8714-1ca61e615743",
+   "metadata": {},
+   "source": [
+    "## Travel agent example"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "34cd131b-f0c2-4b69-887f-2cbd5afb14a7",
+   "metadata": {},
+   "source": [
+    "In this example we will build a team of travel assistant agents that can communicate with each other.\n",
+    "\n",
+    "We will create 2 agents:\n",
+    "\n",
+    "* `travel_advisor`: can help with travel destination recommendations. Can ask `hotel_advisor` for help.\n",
+    "* `hotel_advisor`: can help with hotel recommendations. Can ask `travel_advisor` for help.\n",
+    "\n",
+    "This is a fully-connected network - every agent can talk to any other agent. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fedc9ed0-e90c-4ee1-a7c6-f5af3c634a7b",
+   "metadata": {},
+   "source": [
+    "First, let's create some of the tools that the agents will be using:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "7e31f258-ec28-4020-b86d-c91dfa9a3bfc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import random\n",
+    "from typing_extensions import Literal\n",
+    "from langchain_core.tools import tool\n",
+    "\n",
+    "\n",
+    "@tool\n",
+    "def get_travel_recommendations():\n",
+    "    \"\"\"Get recommendation for travel destinations\"\"\"\n",
+    "    return random.choice([\"aruba\", \"turks and caicos\"])\n",
+    "\n",
+    "\n",
+    "@tool\n",
+    "def get_hotel_recommendations(location: Literal[\"aruba\", \"turks and caicos\"]):\n",
+    "    \"\"\"Get hotel recommendations for a given destination.\"\"\"\n",
+    "    return {\n",
+    "        \"aruba\": [\n",
+    "            \"The Ritz-Carlton, Aruba (Palm Beach)\"\n",
+    "            \"Bucuti & Tara Beach Resort (Eagle Beach)\"\n",
+    "        ],\n",
+    "        \"turks and caicos\": [\"Grace Bay Club\", \"COMO Parrot Cay\"],\n",
+    "    }[location]\n",
+    "\n",
+    "\n",
+    "@tool(return_direct=True)\n",
+    "def transfer_to_hotel_advisor():\n",
+    "    \"\"\"Ask hotel advisor agent for help.\"\"\"\n",
+    "    return \"Successfully transferred to hotel advisor\"\n",
+    "\n",
+    "\n",
+    "@tool(return_direct=True)\n",
+    "def transfer_to_travel_advisor():\n",
+    "    \"\"\"Ask travel advisor agent for help.\"\"\"\n",
+    "    return \"Successfully transferred to travel advisor\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d8519a32-d23b-48b0-bd18-74f8c0dacf58",
+   "metadata": {},
+   "source": [
+    "!!! note \"Transfer tools\"\n",
```

Comment:
This is a good callout, should be in the previous guide too (multi-agent-multi-turn-convo-functional.ipynb)